app/client/src/features/events/Questions/QuestionActions/DeleteButton.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Button } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { graphql, useMutation, useFragment } from 'react-relay';
+import { DeleteButtonFragment$key } from '@local/__generated__/DeleteButtonFragment.graphql';
+import { DeleteButtonMutation } from '@local/__generated__/DeleteButtonMutation.graphql';
+import { useSnack } from '@local/features/core/useSnack';
+
+const DELETE_QUESTION_FRAGMENT = graphql`
+    fragment DeleteButtonFragment on EventQuestion {
+        id
+        isVisible
+        position
+    }
+`;
+
+const DELETE_QUESTION_MUTATION = graphql`
+    mutation DeleteButtonMutation($input: DeleteQuestion!) {
+        deleteQuestion(input: $input) {
+            isError
+            message
+            body {
+                cursor
+                node {
+                    id
+                }
+            }
+        }
+    }
+`;
+
+interface Props {
+    fragmentRef: DeleteButtonFragment$key;
+    className?: string;
+}
+
+export function DeleteButton({ className = undefined, fragmentRef }: Props) {
+    const { id: questionId, position } = useFragment(DELETE_QUESTION_FRAGMENT, fragmentRef);
+    const [commit] = useMutation<DeleteButtonMutation>(DELETE_QUESTION_MUTATION);
+    const { displaySnack } = useSnack();
+
+    function handleClick() {
+        commit({
+            variables: {
+                input: {
+                    questionId,
+                    isVisible: false,
```

Comment:
Is the goal to eventually treat this like a toggle?

Reply:
Yeah I figured making it a toggle would allow for the restoration of removed questions if we ever want to add that feature down the line.